PcAtChipsetPkg/PcatRealTimeClockRuntimeDxe/PcRtc.c

```diff
@@ -638,6 +639,9 @@ PcRtcSetTime (
   RegisterB.Bits.Set = 1;
   RtcWrite (RTC_ADDRESS_REGISTER_B, RegisterB.Data);
 
+  RegisterA.Data    = RtcRead (RTC_ADDRESS_REGISTER_A);

```

Comment:
This code needs a comment block to explain what a Dv value of 0x07 means and why it is safe to update the RTC when set to value 0x07.  Before this change, the .Dv field is never modified in the RTC.